src/bot.js

```diff
@@ -674,7 +674,12 @@ class Bot extends EventEmitter {
 
 
   stopSong(callback) {
-    const rq = {api: 'room.stop_song', roomid: this.roomId, current_song: this.currentSongId};
+    const rq = {
+	  api: 'room.stop_song',
+	  djid: owner.bot.currentDjId,
```

Comment:
`owner` ? what is that variable ?

Reply:
My apologies. I had it formatted on one line, and decided to copy and paste the multiline version from my code - and missed updating that reference.